rootfs/usr/local/bin/aws-config-setup

```diff
@@ -21,6 +21,27 @@ if [ -z "${AWS_ROOT_ACCOUNT_ID}" ]; then
 	exit 1
 fi
 
+if [ ! -d "/localhost/.awsvault/" ] || [ ! -w "/localhost/.awsvault/" ]; then
+	echo "/localhost/.awsvault/ is not writable"
+	exit 1
+fi
+
+if [ -d "/localhost/.awsvault/keys/" ] && [ ! -w "/localhost/.awsvault/" ]; then
```

Comment:
We want `aws-vault` to create this.

Reply:
`aws-vault` [does create that directory](https://github.com/cloudposse/geodesic/blob/258ca72a1f81a086d7e1bfbf3b8c44ee45123fb2/rootfs/etc/profile.d/aws-vault.sh#L56-L57). But it runs as a hook, and `aws-config-setup` runs on the command line, so there's always a chance of a disconnect.